components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -0,0 +1,602 @@
+	local playerNumber = playerRaceBreakDown.playerNumber or playerBreakDownEvent.playerNumber
+	infobox:header('Player Breakdown', playerNumber)
+	infobox:cell('Number of players', playerNumber)
+	infobox:fcell(CustomLeague._playerBreakDownDisplay(playerRaceBreakDown.display))
+	infobox:fcell(CustomLeague._playerBreakDownDisplay(playerBreakDownEvent.display))
+	Variables.varDefine('tournament_playerNumber', playerNumber)
+
+	--teams section
+	if args.team_number or (not String.isEmpty(args.team1)) then
+		Variables.varDefine('is_team_tournament', 1)
+		infobox:header('Teams', true)
+	end
+	infobox:cell('Number of teams', args.team_number)
+	if not String.isEmpty(args.team1) then
+		infobox:header('Teams', true)
+		local teams = CustomLeague:_makeBasedListFromArgs(args, 'team')
+		infobox	:centeredCell(unpack(teams))
+	end
+
+	--maps
+	if not String.isEmpty(args.map1) then
+		infobox:header('Maps', true)
+		local maps = CustomLeague:_makeBasedListFromArgs(args, 'map')
+		infobox:centeredCell(unpack(maps))
+	elseif not String.isEmpty(args['2map1']) then
+		infobox:header(args['2maptitle'] or '2v2 Maps', true)
+		local maps = CustomLeague:_makeBasedListFromArgs(args, '2map')
+		infobox:centeredCell(unpack(maps))
+	elseif not String.isEmpty(args['3map1']) then
+		infobox:header(args['3maptitle'] or '3v3 Maps', true)
+		local maps = CustomLeague:_makeBasedListFromArgs(args, '3map')
+		infobox:centeredCell(unpack(maps))
+	end
+
+	return infobox
+end
+
+function CustomLeague._playerBreakDownEvent(args)
+	local playerBreakDown = {}
+	local codeS = tonumber(args.code_s_number or 0) or 0
+	local codeA = tonumber(args.code_a_number or 0) or 0
+	local premier = tonumber(args.premier_number or 0) or 0
+	local challenger = tonumber(args.challenger_number or 0) or 0
+	local playerNumber = codeS + codeA + premier + challenger
+
+	if playerNumber > 0 then
+		playerBreakDown.playerNumber = playerNumber
+		playerBreakDown.display = {}
+		if codeS > 0 then
+			playerBreakDown.display[#playerBreakDown.display + 1] = _SICON .. ' ' .. codeS
+		end
+		if codeA > 0 then
+			playerBreakDown.display[#playerBreakDown.display + 1] = _AICON .. ' ' .. codeA
+		end
+		if premier > 0 then
+			playerBreakDown.display[#playerBreakDown.display + 1] = _PICON .. ' ' .. premier
+		end
+		if challenger > 0 then
+			playerBreakDown.display[#playerBreakDown.display + 1] = _CICON .. ' ' .. challenger
+		end
+	end
+	return playerBreakDown
+end
+
+function CustomLeague._playerRaceBreakDown(args)
+	local playerBreakDown = {}
+	local playerNumber = tonumber(args.player_number or 0) or 0
+	local zergNumber = tonumber(args.zerg_number or 0) or 0
+	local terranNumbner = tonumber(args.terran_number or 0) or 0
+	local protossNumber = tonumber(args.protoss_number or 0) or 0
+	local randomNumber = tonumber(args.random_number or 0) or 0
+	if playerNumber == 0 then
+		playerNumber = zergNumber + terranNumbner + protossNumber + randomNumber
+	end
+
+	if playerNumber > 0 then
```

Comment:
```suggestion
		playerNumber = zergNumber + terranNumbner + protossNumber + randomNumber
	elseif playerNumber > 0 then
```

Reply:
can't use `elseif` here
the first if just says determine the number by the race numbers if it wasn't set
the 2nd if then prepares stuff for the display if the number (that maybe got adjusted in the first if) is > 0

Reply:
Oh, misread